packages/react-core/src/components/Form/Form.tsx

```diff
@@ -13,14 +13,17 @@ export interface FormProps extends React.HTMLProps<HTMLFormElement> {
   isWidthLimited?: boolean;
   /** Sets a custom max-width for the form. */
   maxWidth?: string;
+  /** Forwarded ref */
```

Comment:
Could you update this comment to be `/** @hide Forwarded ref **/` since it's a prop that is only used internally and doesn't need to be exposed in the docs?